using_openshift/executing_remote_commands.adoc

```diff
@@ -0,0 +1,74 @@
+= Executing Remote Commands
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+You can use the CLI to execute remote commands in a container. This allows you to run general Linux commands for routine operations in the container.
+
+== Basic Usage
+Support for remote container command execution is built into
+link:../using_openshift/cli.html#common-cli-operations[the CLI]:
+
+****
+`$ osc exec -p _<Pod>_ [-c _<Container>_] _<Command>_ _[<Arg_1> ... <Arg_N>]_`
+****
+
+For example:
+
+====
+
+----
+$ osc exec -p mypod date
+Thu Apr  9 02:21:53 UTC 2015
+----
+====
+
+== Protocol
+Clients initiate the execution of a remote command in a container by issuing a
+request to the Kubernetes API server:
+
+****
+`/proxy/minions/_<Node_Name>_/exec/_<Namespace>_/_<Pod>_/_<Container>_?command=_<Command>_`
```

Comment:
This URL will most likely change before GA, assuming we get our bastion/proxy implemented. cc @csrwng